Deferred/Deferred.swift

```diff
@@ -146,11 +224,26 @@ extension Deferred {
 }
 
 extension Deferred {
+    /**
+    Composes the recieving value with another.
```

Comment:
I just noticed that this isn't quite the same as `all([one, other])` because it returns a tuple rather than an array. :\